Dockerfile_base_image

```diff
@@ -1,7 +1,8 @@
 # syntax = tonistiigi/dockerfile:runmount20180618
 FROM ubuntu:18.04
-RUN --mount=target=/polycube cp -r /polycube /tmp/polycube && \
-cd /tmp/polycube && \
+RUN --mount=target=/polycube \
```

Comment:
Maybe you can use `COPY scripts/pre-requirements.sh /pre-requirements.sh` instead of mounting the whole polycube folder?